source/features/tags-dropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+
+import select from 'select-dom';
+
+import * as api from '../github-helpers/api';
+
+import features from '../feature-manager';
+
+const gql = `
+	repository() {
+		refs(refPrefix: "refs/tags/", last: 100) {
+			nodes {
+				name
+			}
+		}
+	}
+`;
+
+async function init(): Promise<false | void> {
+	const {repository} = await api.v4(gql);
+	const tags = repository.refs.nodes as Array<{name: string}>;
+	if (tags.length === 0) {
+		return false;
+	}
+
+	const searchField = select('input#release-filter')!;
+	searchField.after(
+		<datalist id="rgh-tags-dropdown">
+			{tags.reverse().map(tag => <option value={tag.name}/>)}
+		</datalist>,
+	);
+	searchField.setAttribute('list', 'rgh-tags-dropdown');
+}
+
+void features.add(import.meta.url, {
+	include: [
+		pageDetect.isReleasesOrTags,
```

Comment:
Related:

- https://github.com/refined-github/github-url-detection/issues/168

Reply:
Then this should be `isReleases`? Because the search box only exists there

Reply:
> Then this should use isReleases? Because the search box only exists there


Done

> Also previous tags-dropdown works on tags page, should this one?

The issue is that the native field SEARCHES the releases notes, which means it does nothing on tags. See https://github.com/python/cpython/releases?q=v3&expanded=true which is empty even if there are plenty "v3" tags

So if we were to add this feature there, we'd have to reinvent the feature to _not_ use `datalist`. Not worth it in this PR (or maybe ever)